tools/PI/DevHome.PI/ViewModels/AppDetailsPageViewModel.cs

```diff
@@ -27,6 +35,9 @@ public partial class AppDetailsPageViewModel : ObservableObject
     [ObservableProperty]
     private Visibility _processRunningParamsVisibility = Visibility.Collapsed;
 
+    [ObservableProperty]
+    private Visibility _processPackageVisibility = Visibility.Collapsed;
```

Comment:
NIT: Change this to a bool.

Reply:
I understand that the other members are Visibility.  I prefer to start with bool and slowly convert existing visibilities in the future.

Reply:
Yes, I looked at this, but made it a Visibility to be consistent with the others. I've filed #3455 to review all these across PI and consider a BoolToVisibilityConverter instead.